src/raven_clj/ring.clj

```diff
@@ -2,9 +2,15 @@
   (:require [raven-clj.core :refer [capture]]
             [raven-clj.interfaces :refer [http stacktrace]]))
 
+;; Limit the size of the extra contextual data
+;; https://docs.sentry.io/clientdev/data-handling/
+(defn- truncate-extra-str [text]
+  (subs text 0 (min (count text) 4096)))
+
 (defn capture-error [dsn req ^Throwable error extra app-namespaces http-alter-fn]
   (future (capture dsn (-> (merge extra
-                                  {:message (.getMessage error)})
+                                  {:message (.getMessage error)
+                                   :extra {:ex-data (truncate-extra-str (str (ex-data error)))}})
```

Comment:
@marcomorain I think you may send structured data but in this case that would complicate ensuring that you stay under 4096 characters. If Sentry's UI supports structured data and you have a good idea how to ensure the character limits that would probably be worth a PR 👍